zyte_api/_retry.py

```diff
@@ -177,3 +177,62 @@ def build(self) -> AsyncRetrying:
 
 
 zyte_api_retrying: AsyncRetrying = RetryFactory().build()
+
+
+def is_maybe_temporary_error(exc: BaseException) -> bool:
+    return (
+        isinstance(exc, RequestError)
+        and exc.status >= 500
+        and exc.status not in {503, 520}
+    )
+
+
+class ConservativeRetryFactory(RetryFactory):
```

Comment:
Should we call it "aggressive retrying" instead of "conservative"?

Reply:
Yes. Originally I was going with “greedy”, but I was unsure it was fit for a retry policy.